### PR TITLE
Added figures and assembly instructions

### PR DESCRIPTION
**Pull request by arunumd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
-# cabot_design
\ No newline at end of file
+# Cabot_design
```
Comment (daisukes) [inline README.md]: change here to "Cabot Design" and could you add one sentence about this repo?

Comment (daisukes) [reply README.md]: Also we need to add a link to handle Arduino code repo (also need to make it public)

**Review by daisukes — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (arunumd) [reply README.md]: Please check now

**Comment by daisukes** [comment]
close #2

**Pull request merged**